src/arbitrary/_internals/builders/AnyArbitraryBuilder.ts

```diff
@@ -106,7 +121,7 @@ export function anyArbitraryBuilder(constraints: QualifiedObjectConstraints): Ar
 
   return letrec((tie) => ({
     anything: oneof(
-      { maxDepth, depthFactor },
+      { maxDepth, depthFactor, depthIdentifier },
```

Comment:
Should be part of another PR. Before going into that change should even enrich the possible inputs allowed via depthIdentifier. It should accept a real depthIdentifier and not only an identifier leading to it.